requirements.txt

```diff
@@ -7,3 +7,4 @@ setuptools==74.1.1
 requests~=2.32.3
 python-dotenv==1.0.1
 colorama~=0.4.6
+django-q2==1.6.2
```

Comment:
django-q2 is no longer a requirement, users must add this line themselves or install it directly in their environment if they want to use django-q2 hence the conditional install line in the dockerfile.